tests/hotplug/cpu.go

```diff
@@ -408,3 +446,18 @@ func patchWorkloadUpdateMethodAndRolloutStrategy(kvName string, virtClient kubec
 		return err
 	}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 }
+
+// The VMI is assumed to be already logged-in.
+func getGuestVirtualCpus(vmi *v1.VirtualMachineInstance) (int, error) {
+	res, err := console.RunCommandAndStoreOutput(vmi, "nproc --all", time.Second*30)
```

Comment:
nit: Please consider adding a link to https://man7.org/linux/man-pages/man1/nproc.1.html to the the commit message.

Reply:
done